build/teamcity-jepsen.sh

```diff
@@ -72,8 +72,10 @@ for test in "${tests[@]}"; do
         ssh -o "ServerAliveInterval=60" -o "StrictHostKeyChecking no" -i "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}" "${testcmd}"
         if [ $? -ne 0 ]; then
             # Test failed: grab everything.
-            echo "Test failed. Grabbing logs..."
-            scp -o "StrictHostKeyChecking no" -ri "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}:jepsen/cockroachdb/store/latest" "${artifacts_dir}"
+            echo "Test failed. Grabbing all logs..."
+            archive_path="jepsen/cockroachdb/store/failure-logs.tgz"
+            ssh -o "StrictHostKeyChecking no" -ri "$HOME/.ssh/${KEY_NAME}" "ubuntu@${controller}" "tar -czf ${archive_path} jepsen/cockroachdb/store/latest"
```

Comment:
Testing on a sample payload shows that z gives 13 mb and j gives 10, but j is more than twice as slow - so I'll just leave this alone.